src/classes/CCON_Faculty_TEST.cls

```diff
@@ -439,8 +483,8 @@ public with sharing class CCON_Faculty_TEST {
     }
 
     /*********************************************************************************************************
-    * @description Verifies that when a Course Connection is removed the related Affiliation is also removed.
```

Comment:
This was just a bad copy/pasta so I fixed it.

Reply:
thanks